python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -284,6 +304,7 @@ def measure(self, measRecord: lsst.afw.table.SourceRecord, exposure: afwImage.Ex
         if psf is None:
             raise measBase.FatalAlgorithmError("No PSF in exposure")
 
+        sigmas = self.config.sigmas.list() + ["PsfFlux"]*self.config.doPsfPhotometry
```

Comment:
While this is fine, it's generally better (and arguably more easily readable, even if it's two lines longer) to just do `if self.config.doPsfPhotometry: sigmas += ["PsfFlux"]`.

Reply:
I was trying to avoid `if` blocks but I can make this change.

Reply:
This is abstracted away now. I will retain the same logic, but it does not break the flow in the `measure` routine.